.evergreen/template_ssdlc_compliance_report.md

```diff
@@ -0,0 +1,82 @@
+# ${product_name} SSDLC compliance report
+
+This report is available at
+<https://d-9067613a84.awsapps.com/start/#/console?account_id=857654397073&role_name=Drivers.User&destination=https%3a%2f%2fus-west-1.console.aws.amazon.com%2fs3%2fobject%2fjava-driver-release-assets%3fregion%3dus-west-1%26bucketType%3dgeneral%26prefix%3d${product_name}%2f${product_version}%2fssdlc_compliance_report.md>.
+
+<table>
+  <tr>
+    <th>Product name</th>
+    <td><a href="https://github.com/mongodb/mongo-java-driver">${product_name}</a></td>
+  </tr>
+  <tr>
+    <th>Product version</th>
+    <td>${product_version}</td>
+  </tr>
+  <tr>
+    <th>Report date, UTC</th>
+    <td>${report_date_utc}</td>
+  </tr>
+</table>
+
+## Release creator  
+
+This information is available in multiple ways:
+
+<table>
+  <tr>
+    <th>Evergreen</th>
+    <td>
+        Go to
+        <a href="https://evergreen.mongodb.com/waterfall/mongo-java-driver?bv_filter=Publish%20Release">
+            https://evergreen.mongodb.com/waterfall/mongo-java-driver?bv_filter=Publish%20Release</a>,
+        find the build triggered from Git tag <code>r${product_version}</code>, see who authored it.
+    </td>
+  </tr>
+  <tr>
+    <th>Papertrail, human-readable</th>
+    <td>
+        Go to
+        <a href="https://papertrail.devprod-infra.prod.corp.mongodb.com/product-version?product=${product_name}&version=${product_version}">
+            https://papertrail.devprod-infra.prod.corp.mongodb.com/product-version?product=${product_name}&version=${product_version}</a>,
+        look at the value in the "Submitter" column.
```

Comment:
@dylrich are we comfortable with this?

Reply:
If we're going to include these links in the report, maybe we should pull their templates from Evergreen project properties instead of checking them in to the repo.

Reply:
The UI has no long-term stability guarantees and is intended purely for debugging. It's not an Official(tm) product we support. We have not received direction on an official way to serve Papertrail data yet. I'd be more comfortable with language that says something more vague about "see Papertrail"

Reply:
Done in https://github.com/mongodb/mongo-java-driver/pull/1405/commits/98c51138c7bf48d56678ee53ad3d387573960c46.